ansible_collections/arista/avd/plugins/plugin_utils/schema/avdschematools.py

```
@@ -138,6 +135,17 @@ def handle_validation_exceptions(self, exceptions: Generator, mode: str) -> int:
             if not isinstance(exception, AristaAvdError):
                 continue
 
+            if isinstance(exception, AvdDeprecationWarning):
```

so here the choice is to not be able to disable these warnings and to not count them?

---

The depraction warnings are raised using Ansible's deprecation tooling in display instead of showing them together with "conversion warnings".

---

I have added a comment